operators/multiclusterobservability/manifests/base/config/metrics_allowlist.yaml

```diff
@@ -300,3 +300,53 @@ data:
         expr: sum(rate(node_network_receive_drop_total{job=\"node-exporter\", device!=\"lo\"}[1m])) without(device)
       - record: instance:node_network_transmit_drop_excluding_lo:rate1m
         expr: sum(rate(node_network_transmit_drop_total{job=\"node-exporter\", device!=\"lo\"}[1m])) without(device)
+  metrics_rules.yaml: |
+    - name: HighCPUUsage
+      expr: cluster:container_cpu_usage:ratio * 100 > 80
+      for: 2m
+      metrics_list:
+        container_cpu_cfs_periods_total
+        container_cpu_cfs_throttled_periods_total
+        kube_pod_container_resource_limits
+        kube_pod_container_resource_requests
+        kube_resourcequota
+        mixin_pod_workload
+        namespace_workload_pod:kube_pod_owner:relabel
+        node_namespace_pod_container:container_cpu_usage_seconds_total:sum_irate
+        node_namespace_pod_container:container_cpu_usage_seconds_total:sum_rate
+
+    - name: HighMemoryUsage
+      expr: cluster:memory_usage:ratio * 100 > 80
+      for: 2m
+      metrics_list:
+        - container_memory_cache
+        - container_memory_rss
+        - container_memory_swap
+        - container_memory_working_set_bytes
+        - kube_pod_container_resource_limits
+        - kube_pod_container_resource_requests
+
+    - name: SNOHighCPUUsage
+      expr: cluster:container_cpu_usage:ratio{cluster_type\="SNO\"} * 100 > 80
```

Comment:
cluster_type label is not available in managed cluster side and cannot be used in  collect rule. 
Need to think out a way for this
one option is to add a "selector" field like
```
selector:
  clusterType: "SNO"
```